app/main/views/magazines.py

```diff
@@ -1,22 +1,17 @@
+from flask import current_app
+
 from app import api_client
 from app.main import main
 from app.main.views import render_page
 
 
 @main.route('/magazines')
-@main.route('/magazines/<int:page>')
-def magazines(page=0):
-    magazines = api_client.get_magazines()
-
-    start = page * 5
-    end = (page + 1) * 5
-    next_page = page + 1
-
-    if end > len(magazines):
-        next_page = None
+def magazines():
+    magazines = None
+    if not current_app.config["SHOW_RESOURCE_MAINTENANCE"]:
+        magazines = api_client.get_magazines()
 
     return render_page(
         'views/magazines.html',
```

Comment:
Render the magazine template according to the environment.

Reply:
Is there an example of how to do this?

Reply:
You should be able to switch the template based on the `use_sim_data` function, something like this -

```
if use_sim_data():
  return render_page('views/new_magazines.html', ...)
else:
  return render_page('views/magazines.html', ...)
```